app/routes/goal_routes.py

```diff
@@ -1 +1,83 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, Response, request, jsonify
+import requests
+from app.models.goal import Goal
+from .route_utilities import validate_model, create_model, get_models_with_filters, validate_task_ids, get_tasks_from_ids
+from ..db import db
+
+bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+
+@bp.post("")
+def create_goal():
+    request_body = request.get_json()
+    return create_model(Goal, request_body)
+
+@bp.get("")
+def get_all_goals():
+    return get_models_with_filters(Goal,request.args)
+
+@bp.get("/<id>")
+def get_one_goal(id):
+    goal = validate_model(Goal, id)
+    return goal.to_dict()
```

Comment:
Very concise route functions! Don't forget we can use whitespace to distinguish different logic blocks in our functions. 
```suggestion
@bp.post("")
def create_goal():
    request_body = request.get_json()
    
    return create_model(Goal, request_body)

@bp.get("")
def get_all_goals():
    return get_models_with_filters(Goal,request.args)

@bp.get("/<id>")
def get_one_goal(id):
    goal = validate_model(Goal, id)
    
    return goal.to_dict()
```